index.html

```diff
@@ -41,16 +41,16 @@ <h2>The Past</h2>
             <h2>Why Did It Have To Be Boxes...</h2>
             
             <div class="boxes">
-                <div class="box">Box 1</div>
-                <div class="box">Box 2</div>
-                <div class="box">Box 3</div>
-                <div class="box">Box 4</div>
-                <div class="box">Box 5</div>
-                <div class="box">Box 6</div>
-                <div class="box">Box 7</div>
-                <div class="box">Box 8</div>
-                <div class="box">Box 9</div>
-                <div class="box">Box 10</div>
+                <div class="box-one">Box 1</div>
+                <div class="box-two">Box 2</div>
+                <div class="box-three">Box 3</div>
+                <div class="box-four">Box 4</div>
+                <div class="box-five">Box 5</div>
+                <div class="box-six">Box 6</div>
+                <div class="box-seven">Box 7</div>
+                <div class="box-eight">Box 8</div>
+                <div class="box-nine">Box 9</div>
+                <div class="box-ten">Box 10</div>
             </div>
```

Comment:
while this is not technically wrong, best practice would be sibling selectors or nth-child